x-pack/plugins/observability/public/typings/data_access_service.d.ts

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
I created it to avoid errors like: 
`Unexpected path "../../../../observability/public/data_access_service" imported in restricted zone. Plugins may only import from top-level public and server modules.eslint(@kbn/eslint/no-restricted-paths)`

Reply:
Doesn't it have to be re-exported through `index.ts` anyway?